alerting/src/main/kotlin/com/amazon/opendistroforelasticsearch/alerting/resthandler/RestDeleteMonitorAction.kt

```diff
@@ -49,12 +52,13 @@ class RestDeleteMonitorAction : BaseRestHandler() {
     @Throws(IOException::class)
     override fun prepareRequest(request: RestRequest, client: NodeClient): RestChannelConsumer {
         val monitorId = request.param("monitorID")
+        log.debug("${request.method()} ${AlertingPlugin.MONITOR_BASE_URI}/$monitorId")
+
         val refreshPolicy = RefreshPolicy.parse(request.param(REFRESH, RefreshPolicy.IMMEDIATE.value))
+        val deleteMonitorRequest = DeleteDestinationRequest(monitorId, refreshPolicy)
 
         return RestChannelConsumer { channel ->
-            val deleteRequest = DeleteRequest(ScheduledJob.SCHEDULED_JOBS_INDEX, monitorId)
-                    .setRefreshPolicy(refreshPolicy)
-            client.delete(deleteRequest, RestStatusToXContentListener(channel))
+            client.execute(DeleteDestinationAction.INSTANCE, deleteMonitorRequest, RestToXContentListener(channel))
```

Comment:
This should be `DeleteMonitorAction` instead of ``DeleteDestinationAction`

Reply:
My bad, fixed it. Thanks.